### ISSUE TITLE
Malformed URL generation query parameters

### ISSUE THREAD

**Issue by karlbateman** [badges/shields#772]
Just spotted that the URL generation you have is producing malformed parameters.
So for example you have the following being produced..

```
[![Travis branch](https://img.shields.io/travis/rust-lang/rust/master.svg?maxAge=2592000?style=plastic)]()
```

Which should be..

```
[![Travis branch](https://img.shields.io/travis/rust-lang/rust/master.svg?maxAge=2592000&style=plastic)]()
```

Parameters after the first `?` should be `&`. As far as I can see, this is effecting the Markdown generator.

![screenshot_shieldsio](https://cloud.githubusercontent.com/assets/6918791/17837632/d1ea7240-67af-11e6-97f2-b7d34991d961.png)

Hope it helps :smile:

**Comment by e-cloud** [comment]
@karlbateman  Could you make a PR for this?

**Comment by karlbateman** [comment]
@e-cloud Sure, what do you need from me?

**Comment by e-cloud** [comment]
fix the logic generating the url 😄

**Comment by karlbateman** [comment]
Ok, I'll fork it.. when I get a mo, I will fix it 🚀

**Comment by karlbateman** [comment]
Could you point me in the direction of the component which is generating the URL?

**Comment by e-cloud** [comment]
I think **[it's here](https://github.com/badges/shields/blob/master/index.html#L1288)** where it starts

**Comment by karlbateman** [comment]
Ta

**Comment by karlbateman** [comment]
@e-cloud Hey, I got working on this and have a fix in place, but I am not sure how to test it as I only have the style parameter being generated.

**Comment by e-cloud** [comment]
You can post a PR first, so that i can know what have been changed. Then, we can find a way for testing.

**Comment by espadrine** [comment]
Thank you for reporting this issue!

I think it was fixed by 9ba76602c1cfdd3c5670c0c100f48b05759a6293.
